test/quantization/test_quantized_op.py

```diff
@@ -2549,33 +2583,17 @@ def test_qconv1d(
             dilation,
             groups,
         )
-        true_conv1d.weight = torch.nn.Parameter(W)
-        true_conv1d.bias = torch.nn.Parameter(bias_float) if use_bias else None
-        true_outp = true_conv1d(X)
-        q_result_ref = torch.quantize_per_tensor(
-            true_outp, scale=Y_scale, zero_point=Y_zero_point,
-            dtype=torch.quint8)
-
-        conv_op = torch.nn.quantized.Conv1d(
-            input_channels,
-            output_channels,
-            kernel,
-            stride,
-            pad,
-            dilation,
-            groups,
+        qconv_prepack = torch.ops.quantized.conv1d_prepack
+        qconv = torch.ops.quantized.conv1d
+        if use_relu:
+            qconv = torch.ops.quantized.conv1d_relu
+        self._test_qconv_impl(
+            qconv, qconv_prepack, true_conv1d, batch_size,
+            input_channels_per_group, (length, ),
+            output_channels_per_group, groups, kernel, [stride], [pad],
+            [dilation], X_scale, X_zero_point, W_scale, W_zero_point,
+            Y_scale, Y_zero_point, use_bias, use_relu, use_channelwise
```

Comment:
For the work to be complete, the following need to be done (can be done in later PRs)
1. Ensure that conv1d quantized module calls these ops + module level tests https://github.com/pytorch/pytorch/blob/master/torch/nn/quantized/modules/conv.py#L224
2. Support for Conv1dRelu module
3. Op level benchmarks for Conv1d (Please align shapes with float benchmark so that we can compare speedup easily) (https://github.com/pytorch/pytorch/blob/master/benchmarks/operator_benchmark/pt/qconv_test.py#L62)
4. Fusion support: https://github.com/pytorch/pytorch/blob/master/torch/quantization/fuse_modules.py#L95 (For Conv1d+Relu, Conv1d+Bn1d+Relu)
5. QAT support for Conv1d+Bn1d (This can be low priority)

Reply:
@raghuramank100 1, 2 and 3 are done (with graph mode support) in this stack. I'll work on the fusion support next.